modules/openy_features/openy_node/modules/openy_node_blog/openy_node_blog.install

```diff
@@ -9,19 +9,14 @@
  * Implements hook_uninstall().
  */
 function openy_node_blog_uninstall() {
-  \Drupal::service('openy.modules_manager')
-    ->removeEntityBundle('node', 'node_type', 'blog');
+  $modules_manager = \Drupal::service('openy.modules_manager');
+  $modules_manager->removeEntityBundle('node', 'node_type', 'blog');
+  $modules_manager->removeModuleConfigs('openy_node_blog');
+  // Delete protected entity_view_display configs.
   $configFactory = \Drupal::configFactory();
   $configs = [
```

Comment:
let's investigate how to automate this as well based on our conversation.

Reply:
Let's add these runners globally on event module uninstall
Is there an Event available in the core?

Reply:
I think we can use `hook_modules_uninstalled` here -  This function differs from `hook_uninstall()` in that it gives all other modules a chance to perform actions when a module is uninstalled, whereas hook_uninstall() is only called on the module actually being uninstalled.